docs/Getting Started Part 2/index.md

```diff
@@ -0,0 +1,323 @@
+# Getting Started Part 2
+
+## Connecting HMI to SDL
+
+WebSocket is the primary means of communicating with the SDL Core component from the vehicle. In a basic example, an HTML5 HMI would use a native WebSocket library to communicate with SDL Core.
+
+The HMI Adapter must:
+
+!!! must
```

Comment:
half of these MUST points start with capitals, since they end with periods maybe they all should